COMMERCE-1594 Wrap persistence calls in transaction invoker

ethanbustad opened the pull request:
https://issues.liferay.com/browse/COMMERCE-1594

Commit:
COMMERCE-1594 Wrap persistence calls in transaction invoker

ethanbustad commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/103/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/103/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test - 0 out of 1 jobs passed in 1 minute 3 seconds 370 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/a6e2e24d981c3088301c2231c407634678b0803d">a6e2e24d981c3088301c2231c407634678b0803d</a></p>0 out of 1 jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/103/"><strike><strong>test-subrepository-acceptance-pullrequest(7.1.x)</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-6.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/103/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-6.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/103/">test-subrepository-acceptance-pullrequest(7.1.x)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>The sender branch SHA could not be found on <strong><a href="https://github.com/ethanbustad/com-liferay-commerce/tree/COMMERCE-1594-3">ethanbustad/COMMERCE-1594-3</a></strong>. The sender branch may have been force pushed or deleted after the pull request test was initiated.</p><pre><code>fatal: Not a valid branch point: 'c025b2eb5bd64fd32f832405b081db0a8b046370'.</code></pre></div></div></li></ol></details></html>

Commit:
COMMERCE-1594 Mark as a temporary fix pending LPS-97955

We'll want to replace this with a local service call once the
service method is published and available to consumers.

ethanbustad commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x-next">7.1.x-next</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/75/">test-subrepository-acceptance-pullrequest(7.1.x)</a></p><p><b>For more details click <a href="https://test-1-13.liferay.com//userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/75/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h1>Validation PASSED. Running batch tests.</h1><p>Build Time: 28 minutes 13 seconds 823 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/a6e2e24d981c3088301c2231c407634678b0803d">a6e2e24d981c3088301c2231c407634678b0803d</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :heavy_check_mark:</li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/85//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>10 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/85//testReport">here</a>.</h5></div></html>

ethanbustad commented:
@marco-leo merging to fix errors in portal

liferay-continuous-integration commented:
<html><h3>:x: ci:test - 3 out of 6 jobs passed in 1 hour 8 minutes 26 seconds 563 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/a6e2e24d981c3088301c2231c407634678b0803d">a6e2e24d981c3088301c2231c407634678b0803d</a></p>3 out of 6 jobs PASSED<div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-13.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/75/"><strike><strong>test-subrepository-acceptance-pullrequest(7.1.x)</strong></strike></a></li><li><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/381/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/382/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-integration-mysql57-jdk8</strong></strike></a></li></ul></div><details><summary><strong>3 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/380/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-assemble-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-dist(7.1.x)/71/">test-subrepository-acceptance-pullrequest-dist(7.1.x)/jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/85/">test-subrepository-acceptance-pullrequest-validation(7.1.x)/jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-13.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/75/jenkins-report.html">here</a>.</h5><hr/><h4>Failures (upstream comparison is not available):</h4><ol><li><div><h5><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/381/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>2 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/381//consoleText">AXIS_VARIABLE=0,label_exp=!master #381</a><ol><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/381//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
Failed session factory verification
java.lang.IllegalStateException: Wrong current transaction manager, current session factory classes metadata: {com.liferay.commerce.product.model.impl.CommerceChannelRelImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CommerceChannelRelImpl), com.liferay.commerce.product.model.impl.CPDefinitionOptionValueRelImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPDefinitionOptionValueRelImpl), com.liferay.commerce.product.model.impl.CPDisplayLayoutImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPDisplayLayoutImpl), com.liferay.commerce.product.model.impl.CPFriendlyURLEntryImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPFriendlyURLEntryImpl), com.liferay.commerce.product.model.impl.CPOptionImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPOptionImpl), com.liferay.commerce.product.model.impl.CommerceCatalogImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CommerceCatalogImpl), com.liferay.commerce.product.model.impl.CPDefinitionLocalizationImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPDefinitionLocalizationImpl), com.liferay.commerce.product.model.impl.CPDefinitionImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPDefinitionImpl), com.liferay.commerce.product.model.impl.CPDefinitionLinkImpl=SingleTableEntityPersister...</code></pre></div></li><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/381//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmoke_Smoke_">LocalFile.PortalSmoke#Smoke</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1562885132996/test-subrepository-acceptance-pullrequest(7.1.x)/75/functional-smoke-tomcat90-mysql57-jdk8/0/0/LocalFile.PortalSmoke_Smoke/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1562885132996/test-subrepository-acceptance-pullrequest(7.1.x)/75/functional-smoke-tomcat90-mysql57-jdk8/0/0/LocalFile.PortalSmoke_Smoke/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-13/1562885132996/test-subrepository-acceptance-pullrequest(7.1.x)/75/functional-smoke-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/382/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-integration-mysql57-jdk8</a></h5><div><h6>Job Results:</h6><p>1774 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/382//consoleText">AXIS_VARIABLE=0,label_exp=!master #382</a><ol><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/382//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
Failed session factory verification
java.lang.IllegalStateException: Wrong current transaction manager, current session factory classes metadata: {com.liferay.commerce.product.model.impl.CommerceChannelRelImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CommerceChannelRelImpl), com.liferay.commerce.product.model.impl.CPDefinitionOptionValueRelImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPDefinitionOptionValueRelImpl), com.liferay.commerce.product.model.impl.CPDisplayLayoutImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPDisplayLayoutImpl), com.liferay.commerce.product.model.impl.CPFriendlyURLEntryImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPFriendlyURLEntryImpl), com.liferay.commerce.product.model.impl.CPOptionImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPOptionImpl), com.liferay.commerce.product.model.impl.CommerceCatalogImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CommerceCatalogImpl), com.liferay.commerce.product.model.impl.CPDefinitionLocalizationImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPDefinitionLocalizationImpl), com.liferay.commerce.product.model.impl.CPDefinitionImpl=SingleTableEntityPersister(com.liferay.commerce.product.model.impl.CPDefinitionImpl), com.liferay.commerce.product.model.impl.CPDefinitionLinkImpl=SingleTableEntityPersister...</code></pre></div></li></ol></div></li><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=1,label_exp=!master/382//consoleText">AXIS_VARIABLE=1,label_exp=!master #382</a><ol><li><div><a href="https://test-1-7.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=1,label_exp=!master/382//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a></div></li></ol></div></li></ol></div></li></ol></details></html>
